gdx/src/com/badlogic/gdx/utils/CharArray.java

```diff
@@ -815,8 +815,8 @@ public CharArray appendAll (Iterator<?> iter) {
 
 	/** Appends each item in an array to this CharArray without any separators. Each object is appended using
 	 * {@link #append(Object)}. */
-	public <T> CharArray appendAll (T... array) {
-		if (array.length > 0) {
+	public final <T> CharArray appendAll (T... array) {
+		if (array != null) {
```

Comment:
Null should probably crash naturally. It's an error unless we think there is some reason passing null should do nothing. If we keep the null check, the parameter should have javadoc or `@Null`.

Reply:
This PR was merged over a month and a half ago.